Dockerfile

```diff
@@ -0,0 +1,21 @@
+# set base image (host OS)
+FROM python:3.8
```

Comment:
Maybe we should use `python:3.8-slim` or even `python:3.8-alpine` to make the final image smaller.
Seems like that would be enough for what is being done.

Reply:
I agree, we've been using `python:3.x-alpine` on `hathor-core` and we're already used to it and have had zero issues related to it.

Reply:
Canged to `python:3.8-slim`. Alpine got too painful. Slim worked fine.